### DISCUSSION TITLE
Bookmark sync between devices

### DISCUSSION THREAD

**Discussion by ghost** [iterate-ch/cyberduck#14752] [Ideas]
As a user that works on multiple devices, it would be great if there was a mechanism to synchronize bookmarks between devices.  As a Mac user, perhaps this could be done with iCloud integration.  Another option would be to allow modification of where the bookmark configurations are stored - so that we can choose an iCloud or OneDrive folder.

**Comment by dkocher** [comment]
In #7076.

**Comment by tatze96** [comment]
You can share the bookmarks between multiple systems with the help of symbolic links until a bookmark-sharing option is implemented natively.

For more information and instructions on sharing bookmarks between different computers, refer to our [wiki page](https://docs.cyberduck.io/cyberduck/bookmarks/#sharing-bookmarks-between-different-computers).
